buildSrc/src/main/groovy/org.lfenergy.compas.scl.data.java-common-conventions.gradle

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2021 Alliander N.V.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+plugins {
+    // Apply the java Plugin to add support for Java.
+    id "java"
+}
+
+group 'org.lfenergy.compas.scl.data'
+version '1.0.0-SNAPSHOT'
+
+repositories {
+    mavenCentral()
+    maven {
+        name = "GitHubPackages"
+        url = uri("https://maven.pkg.github.com/com-pas/*")
```

Comment:
Right now * is fine because all we need all the packages from the CoMPAS Github Packages.
But if there are more hosted packages in the future, you don't need everything.

So maybe good to not use the asterix sign here (Using it as first test was handy, I know haha)!